OracleDatabase/RAC/OracleDNSServer/dockerfiles/latest/named.rfc1912.zones

```diff
@@ -0,0 +1,53 @@
+// named.rfc1912.zones:
```

Comment:
The `named` package should install this file. You should not provide it yourself.